### ISSUE TITLE
Coinmarket - cannot continue to buy Testnet BTC (+ icon missing)

### ISSUE THREAD

**Issue by LukasRada** [trezor/trezor-suite#14136]
### LOW PRIORITY (BUT NEEDS TO BE FIXED)
**Describe the bug**
Some providers are allowing to buy or sell testnet BTC. It is very limited amounts and only after test agreement with them.

**How to reproduce**

1. Go to trade section
1. Chose Testnet BTC (see screenshot 1) to buy with 50 EUR
1. Wait till offers load and then choose to compare all offers
1. See all offers with missing icon (see screenshot 2)
2. Select any offer
3. See disabled input (and receive address with validation error message) (see screenshot 3)

**Expected behavior**

**Screenshots**
1. 
![image](https://github.com/user-attachments/assets/e38bb537-8eb0-4981-91f4-4e760d1044c6)
2.
![image](https://github.com/user-attachments/assets/50970b9b-8a54-4307-b221-b72ac6a38f8f)
3.
![image](https://github.com/user-attachments/assets/02755817-3488-43b7-9ea3-9f949f1451c7)

**Comment by STew790** [comment]
**QA OK**
![image](https://github.com/user-attachments/assets/4e628cbc-9c95-4b03-a87a-7dfa2608cf1a)

**Info:**
 - Suite version: web 24.11.0 (e07113831a84986631ded03555f0ad6651e87f87)
 - Browser: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36
 - OS: Linux x86_64
 - Screen: 1920x1080
 - Device: Trezor T3T1 2.8.3 regular (revision 7f373ae71eca855dd41b4a0fdcc7cadaa13a8281)
 - Transport: BridgeTransport 2.0.33
